stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -212,6 +204,13 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
                 }
             }
         }
+
+        supportFragmentManager.executePendingTransactions()
+        rootView.doOnNextLayout {
+            // Expand sheet only after the first fragment is attached so that it animates in.
+            // Further calls to expand() are no-op if the sheet is already expanded.
+            bottomSheetController.expand()
+        }
```

Comment:
Is there a unit test we can add for this?

Reply:
Added a test to verify that the sheet goes to STATE_EXPANDED on start.